src/lang.ts

```diff
@@ -171,9 +168,12 @@ export function getPropertyDescriptor(object: Object, property: string): Propert
 }
 
 export function isIdentical(a: any, b: any): boolean {
```

Comment:
Yes, once the copying stuff lands in master, I don't see any reason for `src/object.ts` to exist. The only reason the Observer classes were using `object.is` was to prevent a circular reference with `lang`. Since it's all been moved to one module in `observe`, I updated `lang.isIdentical` to be more in line with what `Object.is` does.

As we discussed in the call, the changes for `-0` and `+0` are to support `lang.isIdentical(-0, +0) === false`.